instrumentation/runtime-metrics/library/README.md

```diff
@@ -0,0 +1,270 @@
+# JMX Metric Insight
+
+This subsystem provides a framework for collecting and reporting metrics provided by
+[JMX](https://www.oracle.com/technical-resources/articles/javase/jmx.html) through
+local [MBeans](https://docs.oracle.com/javase/tutorial/jmx/mbeans/index.html)
+available within the instrumented application. The required MBeans and corresponding metrics can be described using a YAML configuration file. The individual metric configurations allow precise metric selection and identification.
+
+The selected JMX metrics are reported using the Java Agent internal SDK. This means that they share the configuration and metric exporter with other metrics collected by the agent and are controlled by the same properties, for example `otel.metric.export.interval` or `otel.metrics.exporter`.
```

Comment:
This documentation is misleading because the component doesn't have to be used with the Java agent. Would be useful to describe it in more general terms, and include an example of usage without the agent.

Reply:
In its current form, this is an integral part of the Java agent and the description is accurate. If this changes in the future, the description will be updated.

Reply:
Resolving, I think it's ok for initial PR and release to focus on the Java agent use case